Makefile

```diff
@@ -142,6 +142,25 @@ frontend-lint: node_modules/.uptodate
 bddtests: python
 	@tox -qe bddtests
 
+PIP_COMPILE := pyenv exec pip-compile
+
+requirements/%.txt: requirements/%.in
+	$(PIP_COMPILE) --allow-unsafe --quiet $(PIP_COMPILE_FLAGS) $<
```

Comment:
`*_FLAGS` is another `Makefile` convention for passing flags to compilers. We use this to pass `--upgrade` and `--upgrade-package` to `pip-compile` when we want to upgrade packages. Having minimal hard-coded flags (`--allow-unsafe` and `--quiet`) is also convention.

About `--allow-unsafe`:

```
  --allow-unsafe / --no-allow-unsafe
                                  Pin packages considered unsafe: distribute,
                                  pip, setuptools.
                                  
                                  WARNING: Future versions of pip-tools will
                                  enable this behavior by default. Use --no-
                                  allow-unsafe to keep the old behavior. It is
                                  recommended to pass the --allow-unsafe now
                                  to adapt to the upcoming change.
```